app/views/boards/show.html.erb

```diff
@@ -1,15 +1,23 @@
 <body>
  <div class="container">
  <table class="block-center">
-     <% 4.times do %>
+     <% 4.times do |row| %>
       <tr class="even">
- 	     <% 8.times do %>
- 	      <td></td>	 
+ 	     <% 8.times do |even_cell| %>
+       <% if Piece.where(file: even_cell + 1, rank: row * 2 + 1)[0]%>
```

Comment:
Ya, I totally get what you're saying. I originally had  it where I used the game_id - 

<% if Piece.where(game_id: 1, file: even_cell + 1, rank: row * 2 + 1)[0]%> etc...  
game_id 1 was the fake game i created 

And in the controller I had 

def show
    @my_game = Game.find(1)
  end

Trying to figure out if I can somehow use the game_id to pull the id of the game... hmm